Makefile

```diff
@@ -15,6 +15,15 @@ install-serving:
 install-eventing:
 	INSTALL_SERVING="false" ./hack/install.sh
 
+install-kafka:
+	INSTALL_SERVING="false" INSTALL_KAFKA="true" ./hack/install.sh
```

Comment:
By default eventing, serving are being installed. Knative Kafka sets Eventing as a hard dependency so we need to enable the latter by default.

Reply:
should we have an „install-all“ ?

including serving and strimzi?

Reply:
Makes sense to me to have just one command to launch everything. I will added it.